docker/src/main/java/org/jclouds/docker/compute/strategy/DockerComputeServiceAdapter.java

```diff
@@ -136,6 +140,14 @@ public DockerComputeServiceAdapter(DockerApi api) {
               .publishAllPorts(true)
               .privileged(true);
 
+      if (templateOptions.getDirectPorts().isPresent()) {
```

Comment:
Maybe it is better to use the docker naming `portBindings` instead of `directPorts`